test/dynamo/test_functions.py

```diff
@@ -1623,7 +1628,7 @@ def f(x, output):
             gm.code.strip(),
             """\
 def forward(self, x_1, output_1):
-    triton_kernel_wrapper_functional_proxy = functools_triton_kernel_wrapper_functional(grid = (5,), kwargs = {'in_ptr0': x_1, 'out_ptr': output_1, 'n_elements': 5, 'BLOCK_SIZE': 16});  x_1 = output_1 = None
+    triton_kernel_wrapper_functional_proxy = torch._higher_order_ops.triton_kernel_wrap.triton_kernel_wrapper_functional(kernel_idx = 3, grid = (5,), kwargs = {'in_ptr0': x_1, 'out_ptr': output_1, 'n_elements': 5, 'BLOCK_SIZE': 16});  x_1 = output_1 = None
```

Comment:
You probably want to cleanup the sidetable after each test runs. Otherwise this number will depend on the number of tests that run before this one

Reply:
Good call. Is there a thing that runs after each test (like a cleanup method that i can add to) or do i need to do that manually?